earthpy/spatial.py

```diff
@@ -377,19 +390,18 @@ def plot_bands(arr, title=None, cmap="Greys_r",
             else:
                 ax.set(title='Band %i' %band)
             ax.set(xticks=[], yticks=[])
-        # This loop clears out the plots for bands 8-9 which are empty
-        # These plots are req by matplotlib when you specify plot rows & cols
+        # This loop clears out the plots for axes which are empty
+        # A matplotlib grid is always x by x. eg: 8 bands with 3 cols
```

Comment:
Not sure I understand this comment - "x by x" would lead me to think that matplotlib grids are always square (x rows, x columns)

Reply:
how do i write this... matplotlib will always create an even grid. not square  but...

if you have 8 bands, 8 plots and specify 3 cols, it will produce 9 axes - the last will be blank. 8 bands, 8 plots, 2 cols ... 8 axes will be produced exactly.

Does that make sense?